django/contrib/sitemaps/__init__.py

```diff
@@ -1,3 +1,4 @@
+import warnings
```

Comment:
I'm not sure if the title should say "Fixed" as this PR only deprecates http as the default protocol but doesn't actually change the protocol to https. Please let me know if I should be making any changes.

Reply:
It is fine to say "Fixed" as the fellows will handle making the actual change when removing the deprecation when it matures.